lrauv_gazebo_plugins/models/turbidity_generator/model.sdf

```diff
@@ -5,9 +5,9 @@
 -->
 <sdf version="1.6">
 
-  <!-- Requires ParticleEmitter2 in ign-gazebo 4.8.0, which will be copied
-    to ParticleEmitter in Ignition G.
-    See https://github.com/ignitionrobotics/ign-gazebo/pull/730 -->
+  <!-- Requires ParticleEmitter2 in gz-sim 4.8.0, which will be copied
+    to ParticleEmitter in Gazebo Ggarden.
```

Comment:
@marcoag nit: 
```suggestion
    to ParticleEmitter in Gazebo Garden.
```

Or back to `Gazebo G.`. I see both in other SDF files below.